### ISSUE TITLE
tag proposal: <podcast:rerun>

### ISSUE THREAD

**Issue by Chris-Hindy** [Podcastindex-org/podcast-namespace#308]
Apologies in advance if this is not how it is done - nothing worse than management trying to be hip amongst the coders.
As an avid listener, I think the most irritating moment is when you start your podcast app, get on the road and realise another bloody rerun has been added to your feed. Now I'm on my bike and forced to waste time listening to something I know rather than learning something new. 

If episodes could be tagged "rerun" and a podcatcher could learn to ignore those...well that would be sweet :).

Delete if inappropriate and I will go back to shouting my frustration through the streets of Copenhagen on my 2-wheeler.

**Comment by theDanielJLewis** [comment]
I, too, get annoyed by reruns _unless_ I never heard the original. For example, the late _Ask Me Another_ did some reruns that were only a few months old and I hated that because I knew those episodes. But their feed doesn't go very far back, so when they did a rerun from a couple years ago, it was completely new to me and I _did_ enjoy that one.

Do you think a rerun indication should include a date? This could allow a podcast app to ignore all reruns newer than N years.

**Comment by daveajones** [comment]
I like this idea. It’s nice to have binary flag type indicators in the items like this that do simple things to make the app behave better. Love it.

**Comment by staceygoers** [comment]
Re-runs traditionally get lower listenership anyway, so curious why we might want to make that _more_ so for the podcaster, by giving apps the chance to suppress them.

Let's say I use a podcasting app that won't display a re-run if I've listened to the original episode. If as a listener, I expect my podcast episode each Thursday ... won't I be confused when I don't see one?

Thinking through use cases here. I'm also wondering if this just couldn't be solved by seasons/episodes not just being numerical values.

**Comment by theDanielJLewis** [comment]
> Let's say I use a podcasting app that won't display a re-run if I've listened to the original episode. If as a listener, I expect my podcast episode each Thursday ... won't I be confused when I don't see one?

Yes, I can see the potential confusion there. It could be grayed out, marked as a rerun, and not auto-downloaded.

But thinking about your work with NPR, I think you might really _want_ the ability for listeners to skip reruns if they want, because it would result in more accurate ad stats.

Take me, for example. My Overcast is set to auto-download all new episodes. But when I see by the title or hear in the intro that the episode is a recent rerun, I delete it without listening further. I was counted as a listener, but I didn't listen at all. Thus, the consumption and ad stats would be inaaccurate.

**Comment by jamescridland** [comment]
Of course, ads are normally different in a rerun than a first-play and there are other reasons for publishers to rerun. I enjoyed a rerun recently from Freakonomics Radio - but to me, it wasn't a rerun, just a new episode I'd not heard before. It was accompanied by a new introduction setting it in context, too.

Now, I *guess* can see the argument for:

`<podcast:rerun guid="https://example.com/this-guid-for-the-original-episode" />`

....where an episode that is materially the same as a previous one could be marked as a rerun. A pod-catcher would note that I've already listened to the original episode, and therefore mark this as listened-to only if I've listened to the original episode. It would also not auto-download it. Or perhaps it might just mark it as "This is a re-run of a show you heard in July 2018". As a listener, that might be quite good.

But, I can also see a conversation with a publisher going like this:

"So, you want me to spend development time on this, and then you want me to add an additional piece of production workflow, so that... we can earn less money and get fewer downloads?"

I would also suggest that relatively few podcast publishers do reruns: so I'm not sure this is good value for effort.

(On a wider thought - wouldn't it be good if a wiki-style index allowed me as a listener to tag this show as a rerun? Or allowed me to tag this show as being "about chickens" or "funny" or "features Ben Affleck", even if the original podcast publisher couldn't be bothered? So we end up with an additional layer of data for all podcast episodes.)

**Comment by theDanielJLewis** [comment]
I _really_ like @jamescridland's GUID suggestion!

But I also agree that, while nice, I think this could be a DOA feature, unfortunately.

> (On a wider thought - wouldn't it be good if a wiki-style index allowed me as a listener to tag this show as a rerun? Or allowed me to tag this show as being "about chickens" or "funny" or "features Ben Affleck", even if the original podcast publisher couldn't be bothered? So we end up with an additional layer of data for all podcast episodes.)

Yes, that would be nice, but there would need to be a some moderation. Otherwise, liberals could mark conservative podcasts as misinformation or disinformation, and conservatives could do the same to liberal podcasts. Any kind of crowd-sourced tagging would have to be only suggestions. Otherwise, it's opening up to abuse (malicious or spamming).

**Comment by Marzal** [comment]
Hi, as per https://github.com/Podcastindex-org/podcast-namespace/discussions/749 and https://github.com/Podcastindex-org/podcast-namespace/discussions/695 we are closing old issues.

Please open a new [Discussions](https://github.com/Podcastindex-org/podcast-namespace/discussions) if you think that the matter it's worth pursuing.
